base/est/src/main/java/org/dogtagpki/est/ESTEngine.java

```diff
@@ -109,4 +124,37 @@ private void initRequestAuthorizer(String filename) throws Throwable {
         requestAuthorizer.start();
     }
 
+    private void initRealm(String filename) throws Throwable {
+        RealmConfig realmConfig = null;
```

Comment:
(nit) should we add a `this.id != null` check here?  So that we can fail with a useful message if `engine.setId()` has not been called.

Reply:
We could but I think this check should be done in the `start` method eventually. An alternative could be to add the parameter to the constructor so the engine will always have the `id`.